### PR TITLE
Update Patches/Core/Stats/Stats.xml

### PR DESCRIPTION
**Pull request by XeoNovaDan**

### CODE REVIEW

File: Patches/Core/Stats/Stats.xml [hunk 1]
```diff
@@ -56,28 +56,40 @@
...
 
 	<Operation Class="PatchOperationReplace">
-		<xpath>*/StatDef[defName="ShootingAccuracy"]/label</xpath>
+		<xpath>*/StatDef[defName="ShootingAccuracyPawn"]/label</xpath>
 		<value>
 			<label>weapon handling</label>
 		</value>
 	</Operation>
 
+	<Operation Class="PatchOperationSequence">
```
Comment (NoImageAvailable) [inline Patches/Core/Stats/Stats.xml]: Why are we running the same operation twice?

Comment (XeoNovaDan) [reply Patches/Core/Stats/Stats.xml]: First operation is PatchOperationTest and second is PatchOperationRemove

Comment (NoImageAvailable) [reply Patches/Core/Stats/Stats.xml]: Ah didn't spot that. Only issue then is this PR should go into the development branch.

Comment (XeoNovaDan) [reply Patches/Core/Stats/Stats.xml]: Alright, corrected

**Pull request closed without merging**